Assets/2_MainScene/S2_Scripts/S2_Gameplay/ParticleCleanup/HitExplosionCleanup.cs

```diff
@@ -17,10 +17,15 @@ public class HitExplosionCleanup : MonoBehaviour {
 
     void Update() { 
         if (!hitObstacle) return; 
-        transform.Translate(Vector3.back * S2_PoolController.Instance.GetSpeed() * Time.deltaTime); 
+        //transform.Translate(Vector3.back * S2_PoolController.Instance.GetSpeed() * Time.deltaTime); 
```

Comment:
If this is meant to be commented out the update function should probably just be removed, we don't need it to be checking for !hitObstacle if the method is empty

Also I can't remember exactly why but the explosion was moving with the object for some reason and not being parented, do the explosions function properly over moderate play time?